chord_metadata_service/chord/ingest/views.py

```diff
@@ -93,3 +85,42 @@ def ingest(request):
     # TODO: Schema validation
     # TODO: Rollback in case of failures
     return Response(status=204)
+
+
+@api_view(["POST"])
+@permission_classes([AllowAny])
+def ingest_dataset(request, dataset_id: str, data_type: str):
+    logger.info(f"Received a {data_type} ingest request for dataset {dataset_id}.")
+
+    if data_type not in WORKFLOW_INGEST_FUNCTION_MAP:
```

Comment:
this variable is a bit misleading - `data_type` actually contains the Workflow ID here, so worth renaming?